index.html

```diff
@@ -75,6 +75,330 @@ <h4>
+              <li><h5>CANOPY</h5></li>
+              <li>
+                <img src="assets/images/icons/Counter.png" 
+                  alt="circle"/>
+              </li>
+              <li><label>End Dev</label></li>
+              <li>
+                <img src="assets/images/icons/Counter.png" 
+                  alt="circle"/>
+              </li>
+              <li><label>2015</label></li>
+            </ul>
+          </div>
+          <div>
+            <p class="project-description">
+              A daily selection of privately personalized reads; 
+              no accounts or sign-ups required.
+            </p>
+          </div>
+          <ul class="project-button-group">
+            <li>
+              <button type="button" onclick="return;">
+                HTML
+              </button>
+            </li>
+            <li>
+              <button type="button" onclick="return;">
+                CSS
+              </button>
+            </li>
+            <li>
+              <button type="button" onclick="return;">
+                JavaScript
+              </button>
+            </li>
+          </ul>
+          <div>
+            <button class="project-button" type="button"
+              onclick="return;">
+              See project
+            </button>
+          </div>
+        </div>
+        <div class="flex-container card">
+          <img class="project-img" 
+            src="assets/images/samples/Snapshoot Portfolio(4).png" 
+            alt="Portfolio 1" />
+          <h2 class="project-title">
+            Multi-Post Stories
+          </h2>
+          <div>
+            <ul class="flex-list project-date-list">
+              <li><h5>CANOPY</h5></li>
+              <li>
+                <img src="assets/images/icons/Counter.png" 
+                  alt="circle"/>
+              </li>
+              <li><label>End Dev</label></li>
+              <li>
+                <img src="assets/images/icons/Counter.png" 
+                  alt="circle"/>
+              </li>
+              <li><label>2015</label></li>
+            </ul>
+          </div>
+          <div>
+            <p class="project-description">
+              A daily selection of privately personalized reads; 
+              no accounts or sign-ups required.
+            </p>
+          </div>
+          <ul class="project-button-group">
+            <li>
+              <button type="button" onclick="return;">
+                HTML
+              </button>
+            </li>
+            <li>
+              <button type="button" onclick="return;">
+                CSS
+              </button>
+            </li>
+            <li>
+              <button type="button" onclick="return;">
+                JavaScript
+              </button>
+            </li>
+          </ul>
+          <div>
+            <button class="project-button" type="button"
+              onclick="return;">
+              See project
+            </button>
```

Comment:
I would suggest that we add more `padding-top` & bottom for the buttons similar to the Figma design

Your design
![image](https://github.com/chudisoft/Portfolio-Microverse/assets/78251927/894bb49e-2b9b-4e89-961b-41b05f3407fb)

Figma
![image](https://github.com/chudisoft/Portfolio-Microverse/assets/78251927/9156a1c8-b72e-4d60-a1b3-8fc674f48df3)